crates/forge/src/cmd/bind.rs

```diff
@@ -52,6 +52,18 @@ pub struct BindArgs {
     #[arg(long, default_value = DEFAULT_CRATE_VERSION, value_name = "VERSION")]
     crate_version: String,
 
+    /// The description of the Rust crate to generate.
+    ///
+    /// This will be added to the package.description field in Cargo.toml.
+    #[arg(long, value_name = "DESCRIPTION")]
+    crate_description: String,
```

Comment:
I think non optional works because I think we can just add an empty description by default